Code/GraphMol/RWMol.cpp

```diff
@@ -875,7 +876,8 @@ void RWMol::batchRemoveAtoms() {
     INT_VECT &oldStereoAtoms = bond->getStereoAtoms();
     if (oldStereoAtoms.size()) {
       for (auto &idx : oldStereoAtoms) {
-        if (oldIndices[idx]) {
+	// We can't trust stereoatom indices so make sure they really exist
+        if (rdcast<unsigned int>(idx) < numAtoms && oldIndices[idx]) {
```

Comment:
perhaps oldIndices.at(idx) would be better here

Reply:
> perhaps oldIndices.at(idx) would be better here

yeah, I would do this